ocs_ci/ocs/uninstall.py

```diff
@@ -212,14 +226,85 @@ def uninstall_ocs():
             .get("storageClassName")
         )
 
+    # step 6: delete storagecluster
+    cleanup_policy = (
+        storage_cluster.get()
+        .get("metadata")
+        .get("annotations")
+        .get("uninstall.ocs.openshift.io/cleanup-policy")
+    )
+
     log.info("Deleting storageCluster object")
     storage_cluster.delete(resource_name=constants.DEFAULT_CLUSTERNAME)
 
+    # step 7+9: check cleanup pods are compleated
+    storage_node_list = get_labeled_nodes(constants.OPERATOR_NODE_LABEL)
+    if cleanup_policy == "delete":
+        # check clean up pods
+        cleanup_pods = [
+            pod for pod in get_all_pods() if "cluster-cleanup-job" in pod.name
+        ]
+        for pod in cleanup_pods:
+            if pod.get().get("status").get("phase") == "Succeeded":
```

Comment:
shouldn't we wait for the job to complete?

Reply:
while/timeout/sleep?